### PR TITLE
fix(ListView): droid lv crash when scrolled

### LINKED ISSUE #13680: [Droid] `ListView` Called detach on an already detached child
### Current behavior

Simply scrolling ListView with items produces Unhandled exception below and unable to scroll up or down any further

Stack trace
```
Java.Lang.IllegalArgumentException: called detach on an already detached child UnoViewHolder{551d385 position=10 id=-1, oldPos=-1, pLpos:-1 tmpDetached} crc6499cc3f8d6dc23bc6.NativeListViewBase{ced9e11 VFED.V... ......ID 0,0-1080,1719}, adapter:crc6499cc3f8d6dc23bc6.NativeListViewBaseAdapter@96665e4, layout:Microsoft.UI.Xaml.Controls.ItemsStackPanelLayout, context:JupiterTrade.Droid.MainActivity
   at Java.Interop.JniEnvironment.InstanceMethods.CallNonvirtualVoidMethod(JniObjectReference instance, JniObjectReference type, JniMethodInfo method, JniArgumentValue* args) in /Users/runner/work/1/s/xamarin-android/external/Java.Interop/src/Java.Interop/obj/Release/net7.0/JniEnvironment.g.cs:line 12324
   at Java.Interop.JniPeerMembers.JniInstanceMethods.InvokeVirtualVoidMethod(String encodedMember, IJavaPeerable self, JniArgumentValue* parameters) in /Users/runner/work/1/s/xamarin-android/external/Java.Interop/src/Java.Interop/Java.Interop/JniPeerMembers.JniInstanceMethods_Invoke.cs:line 75
   at AndroidX.RecyclerView.Widget.RecyclerView.LayoutManager.DetachView(View child) in C:\a\_work\1\s\generated\androidx.recyclerview.recyclerview\obj\Release
et6.0-android\generated\src\AndroidX.RecyclerView.Widget.RecyclerView.cs:line 5185
   at Microsoft.UI.Xaml.Controls.VirtualizingPanelLayout.TryDetachView(View view) in C:\a\1\s\src\Uno.UI\UI\Xaml\Controls\ListViewBase\VirtualizingPanelLayout.Android.cs:line 2001
   at Microsoft.UI.Xaml.Controls.BufferViewCache.DetachAndCacheView(View view, Recycler recycler) in C:\a\1\s\src\Uno.UI\UI\Xaml\Controls\ListViewBase\BufferViewCache.Android.cs:line 160
   at Microsoft.UI.Xaml.Controls.VirtualizingPanelLayout.RemoveTrailingView(GeneratorDirection fillDirection, Recycler recycler, Boolean detachOnly) in C:\a\1\s\src\Uno.UI\UI\Xaml\Controls\ListViewBase\VirtualizingPanelLayout.Android.cs:line 2563
   at Microsoft.UI.Xaml.Controls.VirtualizingPanelLayout.RemoveTrailingLine(GeneratorDirection fillDirection, Recycler recycler, Boolean detachOnly) in C:\a\1\s\src\Uno.UI\UI\Xaml\Controls\ListViewBase\VirtualizingPanelLayout.Android.cs:line 2577
   at Microsoft.UI.Xaml.Controls.VirtualizingPanelLayout.UnfillLayout(GeneratorDirection direction, Int32 offset, Int32 availableExtent, Recycler recycler, State state) in C:\a\1\s\src\Uno.UI\UI\Xaml\Controls\ListViewBase\VirtualizingPanelLayout.Android.cs:line 1711
   at Microsoft.UI.Xaml.Controls.VirtualizingPanelLayout.ScrollByInner(Int32 offset, Recycler recycler, State state) in C:\a\1\s\src\Uno.UI\UI\Xaml\Controls\ListViewBase\VirtualizingPanelLayout.Android.cs:line 1218
   at Microsoft.UI.Xaml.Controls.VirtualizingPanelLayout.ScrollBy(Int32 offset, Recycler recycler, State state) in C:\a\1\s\src\Uno.UI\UI\Xaml\Controls\ListViewBase\VirtualizingPanelLayout.Android.cs:line 1148
   at Microsoft.UI.Xaml.Controls.VirtualizingPanelLayout.ScrollVerticallyBy(Int32 dy, Recycler recycler, State state) in C:\a\1\s\src\Uno.UI\UI\Xaml\Controls\ListViewBase\VirtualizingPanelLayout.Android.cs:line 220
  --- End of managed Java.Lang.IllegalArgumentException stack trace ---
java.lang.IllegalArgumentException: called detach on an already detached child UnoViewHolder{551d385 position=10 id=-1, oldPos=-1, pLpos:-1 tmpDetached} crc6499cc3f8d6dc23bc6.NativeListViewBase{ced9e11 VFED.V... ......ID 0,0-1080,1719}, adapter:crc6499cc3f8d6dc23bc6.NativeListViewBaseAdapter@96665e4, layout:Microsoft.UI.Xaml.Controls.ItemsStackPanelLayout, context:JupiterTrade.Droid.MainActivity
	at androidx.recyclerview.widget.RecyclerView$5.detachViewFromParent(RecyclerView.java:1038)
	at androidx.recyclerview.widget.ChildHelper.detachViewFromParent(ChildHelper.java:284)
	at androidx.recyclerview.widget.RecyclerView$LayoutManager.detachViewInternal(RecyclerView.java:9619)
	at androidx.recyclerview.widget.RecyclerView$LayoutManager.detachView(RecyclerView.java:9592)
	at crc6499cc3f8d6dc23bc6.VirtualizingPanelLayout.n_scrollVerticallyBy(Native Method)
	at crc6499cc3f8d6dc23bc6.VirtualizingPanelLayout.scrollVerticallyBy(VirtualizingPanelLayout.java:67)
	at androidx.recyclerview.widget.RecyclerView.scrollStep(RecyclerView.java:2050)
	at androidx.recyclerview.widget.RecyclerView.scrollByInternal(RecyclerView.java:2149)
	at androidx.recyclerview.widget.RecyclerView.onTouchEvent(RecyclerView.java:3867)
	at android.view.View.dispatchTouchEvent(View.java:15026)
	at android.view.ViewGroup.dispatchTransformedTouchEvent(ViewGroup.java:3115)
	at android.view.ViewGroup.dispatchTouchEvent(ViewGroup.java:2788)
	at Uno.UI.UnoRecyclerView.access$001(UnoRecyclerView.java:10)
	at Uno.UI.UnoRecyclerView$TouchMotionTarget.dispatchToSuper(UnoRecyclerView.java:37)
	at Uno.UI.UnoMotionHelper.dispatchMotionEvent(UnoMotionHelper.java:191)
	at Uno.UI.UnoRecyclerView.dispatchTouchEvent(UnoRecyclerView.java:50)
	at android.view.ViewGroup.dispatchTransformedTouchEvent(ViewGroup.java:3121)
	at android.view.ViewGroup.dispatchTouchEvent(ViewGroup.java:2802)
	at Uno.UI.UnoViewGroup.access$201(UnoViewGroup.java:14)
	at Uno.UI.UnoViewGroup$TouchMotionTarget.dispatchToSuper(UnoViewGroup.java:364)
	at Uno.UI.UnoMotionHelper.dispatchMotionEvent(UnoMotionHelper.java:191)
	at Uno.UI.UnoViewGroup.dispatchTouchEvent(UnoViewGroup.java:377)
	at android.view.ViewGroup.dispatchTransformedTouchEvent(ViewGroup.java:3121)
	at android.view.ViewGroup.dispatchTouchEvent(ViewGroup.java:2802)
	at Uno.UI.UnoViewGroup.access$201(UnoViewGroup.java:14)
	at Uno.UI.UnoViewGroup$TouchMotionTarget.dispatchToSuper(UnoViewGroup.java:364)
	at Uno.UI.UnoMotionHelper.dispatchMotionEvent(UnoMotionHelper.java:191)
	at Uno.UI.UnoViewGroup.dispatchTouchEvent(UnoViewGroup.java:377)
	at android.view.ViewGroup.dispatchTransformedTouchEvent(ViewGroup.java:3121)
	at android.view.ViewGroup.dispatchTouchEvent(ViewGroup.java:2802)
	at Uno.UI.UnoViewGroup.access$201(UnoViewGroup.java:14)
	at Uno.UI.UnoViewGroup$TouchMotionTarget.dispatchToSuper(UnoViewGroup.java:364)
	at Uno.UI.UnoMotionHelper.dispatchMotionEvent(UnoMotionHelper.java:191)
	at Uno.UI.UnoViewGroup.dispatchTouchEvent(UnoViewGroup.java:377)
	at android.view.ViewGroup.dispatchTransformedTouchEvent(ViewGroup.java:3121)
	at android.view.ViewGroup.dispatchTouchEvent(ViewGroup.java:2802)
	at Uno.UI.UnoViewGroup.access$201(UnoViewGroup.java:14)
	at Uno.UI.UnoViewGroup$TouchMotionTarget.dispatchToSuper(UnoViewGroup.java:364)
	at Uno.UI.UnoMotionHelper.dispatchMotionEvent(UnoMotionHelper.java:191)
	at Uno.UI.UnoViewGroup.dispatchTouchEvent(UnoViewGroup.java:377)
	at android.view.ViewGroup.dispatchTransformedTouchEvent(ViewGroup.java:3121)
	at android.view.ViewGroup.dispatchTouchEvent(ViewGroup.java:2802)
	at Uno.UI.UnoViewGroup.access$201(UnoViewGroup.java:14)
	at Uno.UI.UnoViewGroup$TouchMotionTarget.dispatchToSuper(UnoViewGroup.java:364)
	at Uno.UI.UnoMotionHelper.dispatchMotionEvent(UnoMotionHelper.java:191)
	at Uno.UI.UnoViewGroup.dispatchTouchEvent(UnoViewGroup.java:377)
	at crc6499cc3f8d6dc23bc6.ListViewBase.n_dispatchTouchEvent(Native Method)
	at crc6499cc3f8d6dc23bc6.ListViewBase.dispatchTouchEvent(ListViewBase.java:30)
	at android.view.ViewGroup.dispatchTransformedTouchEvent(ViewGroup.java:3121)
	at android.view.ViewGroup.dispatchTouchEvent(ViewGroup.java:2802)
	at Uno.UI.UnoViewGroup.access$201(UnoViewGroup.java:14)
	at Uno.UI.UnoViewGroup$TouchMotionTarget.dispatchToSuper(UnoViewGroup.java:364)
	at Uno.UI.UnoMotionHelper.dispatchMotionEvent(UnoMotionHelper.java:191)
	at Uno.UI.UnoViewGroup.dispatchTouchEvent(UnoViewGroup.java:377)
	at android.view.ViewGroup.dispatchTransformedTouchEvent(ViewGroup.java:3121)
	at android.view.ViewGroup.dispatchTouchEvent(ViewGroup.java:2802)
	at crc64bad6bab841c07a1a.NativeRefreshControl.n_dispatchTouchEvent(Native Method)
	at crc64bad6bab841c07a1a.NativeRefreshControl.dispatchTouchEvent(NativeRefreshControl.java:86)
	at android.view.ViewGroup.dispatchTransformedTouchEvent(ViewGroup.java:3121)
	at android.view.ViewGroup.dispatchTouchEvent(ViewGroup.java:2802)
	at Uno.UI.UnoViewGroup.access$201(UnoViewGroup.java:14)
	at Uno.UI.UnoViewGroup$TouchMotionTarget.dispatchToSuper(UnoViewGroup.java:364)
	at Uno.UI.UnoMotionHelper.dispatchMotionEvent(UnoMotionHelper.java:191)
	at Uno.UI.UnoViewGroup.dispatchTouchEvent(UnoViewGroup.java:377)
	at android.view.ViewGroup.dispatchTransformedTouchEvent(ViewGroup.java:3121)
	at android.view.ViewGroup.dispatchTouchEvent(ViewGroup.java:2802)
	at Uno.UI.UnoViewGroup.access$201(UnoViewGroup.java:14)
	at Uno.UI.UnoViewGroup$TouchMotionTarget.dispatchToSuper(UnoViewGroup.java:364)
	at Uno.UI.UnoMotionHelper.dispatchMotionEvent(UnoMotionHelper.java:191)
	at Uno.UI.UnoViewGroup.dispatchTouchEvent(UnoViewGroup.java:377)
	at android.view.ViewGroup.dispatchTransformedTouchEvent(ViewGroup.java:3121)
	at android.view.ViewGroup.dispatchTouchEvent(ViewGroup.java:2802)
	at Uno.UI.UnoViewGroup.access$201(UnoViewGroup.java:14)
	at Uno.UI.UnoViewGroup$TouchMotionTarget.dispatchToSuper(UnoViewGroup.java:364)
	at Uno.UI.UnoMotionHelper.dispatchMotionEvent(UnoMotionHelper.java:191)
	at Uno.UI.UnoViewGroup.dispatchTouchEvent(UnoViewGroup.java:377)
	at android.view.ViewGroup.dispatchTransformedTouchEvent(ViewGroup.java:3121)
	at android.view.ViewGroup.dispatchTouchEvent(ViewGroup.java:2802)
	at Uno.UI.UnoViewGroup.access$201(UnoViewGroup.java:14)
	at Uno.UI.UnoViewGroup$TouchMotionTarget.dispatchToSuper(UnoViewGroup.java:364)
	at Uno.UI.UnoMotionHelper.dispatchMotionEvent(UnoMotionHelper.java:191)
	at Uno.UI.UnoViewGroup.dispatchTouchEvent(UnoViewGroup.java:377)
	at android.view.ViewGroup.dispatchTransformedTouchEvent(ViewGroup.java:3121)
	at android.view.ViewGroup.dispatchTouchEvent(ViewGroup.java:2802)
	at Uno.UI.UnoViewGroup.access$201(UnoViewGroup.java:14)
	at Uno.UI.UnoViewGroup$TouchMotionTarget.dispatchToSuper(UnoViewGroup.java:364)
	at Uno.UI.UnoMotionHelper.dispatchMotionEvent(UnoMotionHelper.java:191)
	at Uno.UI.UnoViewGroup.dispatchTouchEvent(UnoViewGroup.java:377)
	at android.view.ViewGroup.dispatchTransformedTouchEvent(ViewGroup.java:3121)
	at android.view.ViewGroup.dispatchTouchEvent(ViewGroup.java:2802)
	at Uno.UI.UnoViewGroup.access$201(UnoViewGroup.java:14)
	at Uno.UI.UnoViewGroup$TouchMotionTarget.dispatchToSuper(UnoViewGroup.java:364)
	at Uno.UI.UnoMotionHelper.dispatchMotionEvent(UnoMotionHelper.java:191)
	at Uno.UI.UnoViewGroup.dispatchTouchEvent(UnoViewGroup.java:377)
	at android.view.ViewGroup.dispatchTransformedTouchEvent(ViewGroup.java:3121)
	at android.view.ViewGroup.dispatchTouchEvent(ViewGroup.java:2802)
	at Uno.UI.UnoViewGroup.access$201(UnoViewGroup.java:14)
	at Uno.UI.UnoViewGroup$TouchMotionTarget.dispatchToSuper(UnoViewGroup.java:364)
	at Uno.UI.UnoMotionHelper.dispatchMotionEvent(UnoMotionHelper.java:191)
	at Uno.UI.UnoViewGroup.dispatchTouchEvent(UnoViewGroup.java:377)
	at android.view.ViewGroup.dispatchTransformedTouchEvent(ViewGroup.java:3121)
	at android.view.ViewGroup.dispatchTouchEvent(ViewGroup.java:2802)
	at Uno.UI.UnoViewGroup.access$201(UnoViewGroup.java:14)
	at Uno.UI.UnoViewGroup$TouchMotionTarget.dispatchToSuper(UnoViewGroup.java:364)
	at Uno.UI.UnoMotionHelper.dispatchMotionEvent(UnoMotionHelper.java:191)
	at Uno.UI.UnoViewGroup.dispatchTouchEvent(UnoViewGroup.java:377)
	at android.view.ViewGroup.dispatchTransformedTouchEvent(ViewGroup.java:3121)
	at android.view.ViewGroup.dispatchTouchEvent(ViewGroup.java:2802)
	at Uno.UI.UnoViewGroup.access$201(UnoViewGroup.java:14)
	at Uno.UI.UnoViewGroup$TouchMotionTarget.dispatchToSuper(UnoViewGroup.java:364)
	at Uno.UI.UnoMotionHelper.dispatchMotionEvent(UnoMotionHelper.java:191)
	at Uno.UI.UnoViewGroup.dispatchTouchEvent(UnoViewGroup.java:377)
	at android.view.ViewGroup.dispatchTransformedTouchEvent(ViewGroup.java:3121)
	at android.view.ViewGroup.dispatchTouchEvent(ViewGroup.java:2802)
	at Uno.UI.UnoViewGroup.access$201(UnoViewGroup.java:14)
	at Uno.UI.UnoViewGroup$TouchMotionTarget.dispatchToSuper(UnoViewGroup.java:364)
	at Uno.UI.UnoMotionHelper.dispatchMotionEvent(UnoMotionHelper.java:191)
	at Uno.UI.UnoViewGroup.dispatchTouchEvent(UnoViewGroup.java:377)
	at android.view.ViewGroup.dispatchTransformedTouchEvent(ViewGroup.java:3121)
	at android.view.ViewGroup.dispatchTouchEvent(ViewGroup.java:2802)
	at Uno.UI.UnoViewGroup.access$201(UnoViewGroup.java:14)
	at Uno.UI.UnoViewGroup$TouchMotionTarget.dispatchToSuper(UnoViewGroup.java:364)
	at Uno.UI.UnoMotionHelper.dispatchMotionEvent(UnoMotionHelper.java:191)
	at Uno.UI.UnoViewGroup.dispatchTouchEvent(UnoViewGroup.java:377)
	at android.view.ViewGroup.dispatchTransformedTouchEvent(ViewGroup.java:3121)
	at android.view.ViewGroup.dispatchTouchEvent(ViewGroup.java:2802)
	at Uno.UI.UnoViewGroup.access$201(UnoViewGroup.java:14)
	at Uno.UI.UnoViewGroup$TouchMotionTarget.dispatchToSuper(UnoViewGroup.java:364)
	at Uno.UI.UnoMotionHelper.dispatchMotionEvent(UnoMotionHelper.java:191)
	at Uno.UI.UnoViewGroup.dispatchTouchEvent(UnoViewGroup.java:377)
	at Uno.UI.TouchMotionTarget.dispatchToChild(UnoMotionHelper.java:53)
	at Uno.UI.UnoMotionHelper.dispatchStaticTransformedMotionEventCore(UnoMotionHelper.java:419)
	at Uno.UI.UnoMotionHelper.dispatchStaticTransformedMotionEvent(UnoMotionHelper.java:389)
	at Uno.UI.UnoMotionHelper.dispatchStaticTransformedMotionEvent(UnoMotionHelper.java:331)
	at Uno.UI.UnoMotionHelper.dispatchMotionEvent(UnoMotionHelper.java:187)
	at Uno.UI.UnoViewGroup.dispatchTouchEvent(UnoViewGroup.java:377)
	at android.view.ViewGroup.dispatchTransformedTouchEvent(ViewGroup.java:3121)
	at android.view.ViewGroup.dispatchTouchEvent(ViewGroup.java:2802)
	at Uno.UI.UnoViewGroup.access$201(UnoViewGroup.java:14)
	at Uno.UI.UnoViewGroup$TouchMotionTarget.dispatchToSuper(UnoViewGroup.java:364)
	at Uno.UI.UnoMotionHelper.dispatchMotionEvent(UnoMotionHelper.java:191)
	at Uno.UI.UnoViewGroup.dispatchTouchEvent(UnoViewGroup.java:377)
	at Uno.UI.TouchMotionTarget.dispatchToChild(UnoMotionHelper.java:53)
	at Uno.UI.UnoMotionHelper.dispatchStaticTransformedMotionEventCore(UnoMotionHelper.java:419)
	at Uno.UI.UnoMotionHelper.dispatchStaticTransformedMotionEvent(UnoMotionHelper.java:389)
	at Uno.UI.UnoMotionHelper.dispatchStaticTransformedMotionEvent(UnoMotionHelper.java:331)
	at Uno.UI.UnoMotionHelper.dispatchMotionEvent(UnoMotionHelper.java:187)
	at Uno.UI.UnoViewGroup.dispatchTouchEvent(UnoViewGroup.java:377)
	at android.view.ViewGroup.dispatchTransformedTouchEvent(ViewGroup.java:3121)
	at android.view.ViewGroup.dispatchTouchEvent(ViewGroup.java:2802)
	at Uno.UI.UnoViewGroup.access$201(UnoViewGroup.java:14)
	at Uno.UI.UnoViewGroup$TouchMotionTarget.dispatchToSuper(UnoViewGroup.java:364)
	at Uno.UI.UnoMotionHelper.dispatchMotionEvent(UnoMotionHelper.java:191)
	at Uno.UI.UnoViewGroup.dispatchTouchEvent(UnoViewGroup.java:377)
	at android.view.ViewGroup.dispatchTransformedTouchEvent(ViewGroup.java:3121)
	at android.view.ViewGroup.dispatchTouchEvent(ViewGroup.java:2802)
	at Uno.UI.UnoViewGroup.access$201(UnoViewGroup.java:14)
	at Uno.UI.UnoViewGroup$TouchMotionTarget.dispatchToSuper(UnoViewGroup.java:364)
	at Uno.UI.UnoMotionHelper.dispatchMotionEvent(UnoMotionHelper.java:191)
	at Uno.UI.UnoViewGroup.dispatchTouchEvent(UnoViewGroup.java:377)
	at android.view.ViewGroup.dispatchTransformedTouchEvent(ViewGroup.java:3121)
	at android.view.ViewGroup.dispatchTouchEvent(ViewGroup.java:2802)
	at Uno.UI.UnoViewGroup.access$201(UnoViewGroup.java:14)
	at Uno.UI.UnoViewGroup$TouchMotionTarget.dispatchToSuper(UnoViewGroup.java:364)
	at Uno.UI.UnoMotionHelper.dispatchMotionEvent(UnoMotionHelper.java:191)
	at Uno.UI.UnoViewGroup.dispatchTouchEvent(UnoViewGroup.java:377)
	at android.view.ViewGroup.dispatchTransformedTouchEvent(ViewGroup.java:3121)
	at android.view.ViewGroup.dispatchTouchEvent(ViewGroup.java:2802)
	at Uno.UI.UnoViewGroup.access$201(UnoViewGroup.java:14)
	at Uno.UI.UnoViewGroup$TouchMotionTarget.dispatchToSuper(UnoViewGroup.java:364)
	at Uno.UI.UnoMotionHelper.dispatchMotionEvent(UnoMotionHelper.java:191)
	at Uno.UI.UnoViewGroup.dispatchTouchEvent(UnoViewGroup.java:377)
	at android.view.ViewGroup.dispatchTransformedTouchEvent(ViewGroup.java:3121)
	at android.view.ViewGroup.dispatchTouchEvent(ViewGroup.java:2802)
	at Uno.UI.UnoViewGroup.access$201(UnoViewGroup.java:14)
	at Uno.UI.UnoViewGroup$TouchMotionTarget.dispatchToSuper(UnoViewGroup.java:364)
	at Uno.UI.UnoMotionHelper.dispatchMotionEvent(UnoMotionHelper.java:191)
	at Uno.UI.UnoViewGroup.dispatchTouchEvent(UnoViewGroup.java:377)
	at android.view.ViewGroup.dispatchTransformedTouchEvent(ViewGroup.java:3121)
	at android.view.ViewGroup.dispatchTouchEvent(ViewGroup.java:2802)
	at Uno.UI.UnoViewGroup.access$201(UnoViewGroup.java:14)
	at Uno.UI.UnoViewGroup$TouchMotionTarget.dispatchToSuper(UnoViewGroup.java:364)
	at Uno.UI.UnoMotionHelper.dispatchMotionEvent(UnoMotionHelper.java:191)
	at Uno.UI.UnoViewGroup.dispatchTouchEvent(UnoViewGroup.java:377)
	at android.view.ViewGroup.dispatchTransformedTouchEvent(ViewGroup.java:3121)
	at android.view.ViewGroup.dispatchTouchEvent(ViewGroup.java:2802)
	at Uno.UI.UnoViewGroup.access$201(UnoViewGroup.java:14)
	at Uno.UI.UnoViewGroup$TouchMotionTarget.dispatchToSuper(UnoViewGroup.java:364)
	at Uno.UI.UnoMotionHelper.dispatchMotionEvent(UnoMotionHelper.java:191)
	at Uno.UI.UnoViewGroup.dispatchTouchEvent(UnoViewGroup.java:377)
	at android.view.ViewGroup.dispatchTransformedTouchEvent(ViewGroup.java:3121)
	at android.view.ViewGroup.dispatchTouchEvent(ViewGroup.java:2802)
	at android.view.ViewGroup.dispatchTransformedTouchEvent(ViewGroup.java:3121)
	at android.view.ViewGroup.dispatchTouchEvent(ViewGroup.java:2802)
	at android.view.ViewGroup.dispatchTransformedTouchEvent(ViewGroup.java:3121)
	at android.view.ViewGroup.dispatchTouchEvent(ViewGroup.java:2802)
	at android.view.ViewGroup.dispatchTransformedTouchEvent(ViewGroup.java:3121)
	at android.view.ViewGroup.dispatchTouchEvent(ViewGroup.java:2802)
	at android.view.ViewGroup.dispatchTransformedTouchEvent(ViewGroup.java:3121)
	at android.view.ViewGroup.dispatchTouchEvent(ViewGroup.java:2802)
	at com.android.internal.policy.DecorView.superDispatchTouchEvent(DecorView.java:498)
	at com.android.internal.policy.PhoneWindow.superDispatchTouchEvent(PhoneWindow.java:1896)
	at android.app.Activity.dispatchTouchEvent(Activity.java:4262)
	at crc6467d88aaa7548788d.BaseActivity.n_dispatchTouchEvent(Native Method)
	at crc6467d88aaa7548788d.BaseActivity.dispatchTouchEvent(BaseActivity.java:139)
	at androidx.appcompat.view.WindowCallbackWrapper.dispatchTouchEvent(WindowCallbackWrapper.java:70)
	at com.android.internal.policy.DecorView.dispatchTouchEvent(DecorView.java:456)
	at android.view.View.dispatchPointerEvent(View.java:15285)
	at android.view.ViewRootImpl$ViewPostImeInputStage.processPointerEvent(ViewRootImpl.java:6542)
	at android.view.ViewRootImpl$ViewPostImeInputStage.onProcess(ViewRootImpl.java:6342)
	at android.view.ViewRootImpl$InputStage.deliver(ViewRootImpl.java:5798)
	at android.view.ViewRootImpl$InputStage.onDeliverToNext(ViewRootImpl.java:5855)
	at android.view.ViewRootImpl$InputStage.forward(ViewRootImpl.java:5821)
	at android.view.ViewRootImpl$AsyncInputStage.forward(ViewRootImpl.java:5986)
	at android.view.ViewRootImpl$InputStage.apply(ViewRootImpl.java:5829)
	at android.view.ViewRootImpl$AsyncInputStage.apply(ViewRootImpl.java:6043)
	at android.view.ViewRootImpl$InputStage.deliver(ViewRootImpl.java:5802)
	at android.view.ViewRootImpl$InputStage.onDeliverToNext(ViewRootImpl.java:5855)
	at android.view.ViewRootImpl$InputStage.forward(ViewRootImpl.java:5821)
	at android.view.ViewRootImpl$InputStage.apply(ViewRootImpl.java:5829)
	at android.view.ViewRootImpl$InputStage.deliver(ViewRootImpl.java:5802)
	at android.view.ViewRootImpl.deliverInputEvent(ViewRootImpl.java:8842)
	at android.view.ViewRootImpl.doProcessInputEvents(ViewRootImpl.java:8793)
	at android.view.ViewRootImpl.enqueueInputEvent(ViewRootImpl.java:8762)
	at android.view.ViewRootImpl$WindowInputEventReceiver.onInputEvent(ViewRootImpl.java:8965)
	at android.view.InputEventReceiver.dispatchInputEvent(InputEventReceiver.java:267)
	at android.view.InputEventReceiver.nativeConsumeBatchedInputEvents(Native Method)
	at android.view.InputEventReceiver.consumeBatchedInputEvents(InputEventReceiver.java:247)
	at android.view.ViewRootImpl.doConsumeBatchedInput(ViewRootImpl.java:8922)
	at android.view.ViewRootImpl$ConsumeBatchedInputRunnable.run(ViewRootImpl.java:9056)
	at android.view.Choreographer$CallbackRecord.run(Choreographer.java:1230)
	at android.view.Choreographer$CallbackRecord.run(Choreographer.java:1238)
	at android.view.Choreographer.doCallbacks(Choreographer.java:898)
	at android.view.Choreographer.doFrame(Choreographer.java:823)
	at android.view.Choreographer$FrameDisplayEventReceiver.run(Choreographer.java:1213)
	at android.os.Handler.handleCallback(Handler.java:942)
	at android.os.Handler.dispatchMessage(Handler.java:99)
	at android.os.Looper.loopOnce(Looper.java:201)
	at android.os.Looper.loop(Looper.java:288)
	at android.app.ActivityThread.main(ActivityThread.java:7892)
	at java.lang.reflect.Method.invoke(Native Method)
	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:548)
	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:936)

  --- End of managed Java.Lang.IllegalArgumentException stack trace ---
```

### Expected behavior

No exception with scrolling possibility

### How to reproduce it (as minimally and precisely as possible)

[DroidListViewIssue.zip](https://github.com/unoplatform/uno/files/12655945/DroidListViewIssue.zip)

### Workaround

_No response_

### Works on UWP/WinUI

Yes

### Environment

_No response_

### NuGet package version(s)

Uno.WinUI 4.10.13

### Affected platforms

Android

### IDE

Visual Studio 2022

### IDE version

_No response_

### Relevant plugins

_No response_

### Anything else we need to know?

_No response_

### LINKED ISSUE #13830: GridView/ListView gets stuck on scrolling up or down in Android
### Current behavior

With a mere 100 - 1000 items, GridView/ListView gets stuck after some random scrolling... say scroll fast till bottom, then back to the top, again repeat the same few times and boom!!
Behavior depends a lot on the item template and whether its a grid/list and your overall visual tree. But attached is a simple example that would cause it to get stuck right away, though the item template is simplest possible of any use in production app. Tested in  Pixel 7 Pro, Pixel 2 and Moto E.

Screen Grab below:

https://github.com/unoplatform/uno/assets/16226539/72b4d155-2b50-46f7-ad43-e00d78cc6dfc

### Expected behavior

_No response_

### How to reproduce it (as minimally and precisely as possible)

[UnoApp1.zip](https://github.com/unoplatform/uno/files/12731405/UnoApp1.zip)

### Workaround

_No response_

### Works on UWP/WinUI

None

### Environment

_No response_

### NuGet package version(s)

_No response_

### Affected platforms

_No response_

### IDE

_No response_

### IDE version

_No response_

### Relevant plugins

_No response_

### Anything else we need to know?

_No response_

### PR DESCRIPTION
**Pull request by Xiaoy312**
GitHub Issue (If applicable): closes #13680, closes #13830

## PR Type
What kind of change does this PR introduce?
- Bugfix

## What is the current behavior?
In certain situation, on android, scrolling in a ListView can cause it to freeze from this exception:
> [0:] Java.Lang.IllegalArgumentException: called detach on an already detached child UnoViewHolder{f91d06b position=33 id=-1, oldPos=-1, pLpos:-1 tmpDetached} crc6499cc3f8d6dc23bc6.NativeListViewBase{dbb9976 VFED.V... ........ 0,0-1080,1731}, adapter:crc6499cc3f8d6dc23bc6.NativeListViewBaseAdapter@dbfd702, layout:Microsoft.UI.Xaml.Controls.ItemsStackPanelLayout, context:DroidListViewIssue.Droid.MainActivity

## What is the new behavior?
Should no longer happen.

### <samp>🤖 Generated by Copilot at fa89bf3</samp>

Overloaded `DetachViewFromParent` method and added helper method to fix view holder detachment bug in `NativeListViewBase.Android.cs`. Renamed local variable for clarity.

## PR Checklist
Please check if your PR fulfills the following requirements:
- [ ] Docs have been added/updated which fit [documentation template](https://github.com/unoplatform/uno/blob/master/doc/.feature-template.md) (for bug fixes / features)
- [ ] [Unit Tests and/or UI Tests](https://github.com/unoplatform/uno/blob/master/doc/articles/uno-development/working-with-the-samples-apps.md) for the changes have been added (for bug fixes / features) (if applicable)
- [ ] Validated PR `Screenshots Compare Test Run` results.
- [x] Contains **NO** breaking changes
- [x] Associated with an issue (GitHub or internal) and uses the [automatic close keywords](https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue).
- [x] Commits must be following the [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/#summary) specification.

## Other information
For context, in uno, we are keepting the record of item attaching/detaching via UnoViewHolder.IsDetached from AttachViewToParent & DetachViewFromParent. However, it seems that we have missed a few other methods that could also be used for similar purpose.
The exact setup to reproduce the issue isn't clear. Repro has been unsuccessful in Uno.UI's and Uno.Gallery's SampleApp as well as
a fresh `unoapp` project...

### CODE REVIEW

**Comment by Xiaoy312** [comment]
@jeromelaban for 4x backport

**Comment by jeromelaban** [comment]
@mergifyio backport release/stable/4.10 legacy/4x

**Comment by jeromelaban** [comment]
Blocked by https://github.com/unoplatform/uno/pull/13875, to be re-run after merge.

**Review by jeromelaban — APPROVED** [review]

**Pull request merged**
